config/webpack.browser.config.js

```diff
@@ -2,40 +2,39 @@
 const webpack = require('webpack');
 const path = require('path');
 
-module.exports = function getConfig(env) {
-  const context = path.join(__dirname, '../');
-
-  return {
-    context,
-    entry: {
-      web: ['./src/web/index'],
-    },
-    devtool: 'cheap-module-source-map',
-    plugins: [
-      new webpack.NamedModulesPlugin(),
+module.exports = {
+  context: path.join(__dirname, '../'),
+  entry: {
+    web: ['./src/web/index'],
+  },
+  devtool: 'cheap-module-source-map',
+  optimization: {
+    namedModules: true, // NamedModulesPlugin()
+  },
+  plugins: [
+    new webpack.NamedModulesPlugin(),
+  ],
+  output: {
+    path: path.join(__dirname, '../dist/web'),
+    filename: 'index.js',
+    chunkFilename: '[name].chunk.min.js',
+    publicPath: '../',
+    library: 'huron',
+    libraryTarget: 'commonjs2',
+  },
+  module: {
+    rules: [
+      {
+        enforce: 'pre',
+        test: /\.js$/,
+        exclude: [/\.min\.js$/],
+        use: 'eslint-loader',
+      },
+      {
+        test: /\.js$/,
+        // exclude: /node_modules/,
```

Comment:
🤔 We don't need to exclude `node_modules` for either of these? If not, I'd say I'd say kill this comment.

Reply:
we do, this was me debugging, I'm adding it back, good catch 😅